packages/server/tests/acceptance/ws/subscribe.spec.ts

```diff
@@ -104,6 +106,11 @@ describe('@web-socket Acceptance Tests', async function() {
         wsProvider.destroy();
     });
 
+    this.afterAll(async () => {
+        // Return ENV variables to their original values
+        process.env.WS_MAX_CONNECTION_TTL = '300000';
```

Comment:
nit: if this is supposed to be the original value you should probably cache it first.
The current logic will require this value get updates should the default `WS_MAX_CONNECTION_TTL` ever change.